server/game/cards/events/01/risenfromthesea.js

```diff
@@ -14,6 +14,7 @@ class RisenFromTheSea extends DrawCard {
             handler: context => {
                 context.event.saveCard(context.target);
                 this.controller.attach(this.controller, this, context.target, 'play');
+                this.setCardType('attachment');
```

Comment:
It might be better if this is part of the persistent effect below along with the 'Condition' and 'Terminal' effects. (Ditto Blood Magic Ritual). It may make handling of any attachment blanking in the future more consistent.